k2/csrc/properties_test.cc

```diff
@@ -356,5 +357,41 @@ TEST(Properties, IsConnected) {
     EXPECT_TRUE(is_connected);
   }
 }
+TEST(FsaAlgo, IsAcyclic) {
+  // empty fsa is acyclic
+  {
+    Fsa fsa;
+    bool is_acyclic = IsAcyclic(fsa);
+    EXPECT_FALSE(is_acyclic);
```

Comment:
the comment says
```cpp
// empty fsa is acyclic
 ```

but you use `EXPECT_FALSE`. Should it be `EXPECT_TRUE` ?

Reply:
I addressed this issue in [this commit](https://github.com/danpovey/k2/pull/26/commits/45763678844206c61f81918aab31dce7f46815de).